hrms/hr/doctype/attendance_request/attendance_request.json

```diff
@@ -16,6 +16,7 @@
   "from_date",
   "to_date",
   "half_day",
+  "shift",
```

Comment:
Move the Shift field to LHS for some balance in the layout

<img width="1335" alt="image" src="https://github.com/frappe/hrms/assets/24353136/d71ede10-2487-4790-91df-07d74d3f31b7">